docs/model/datamodel.md

```diff
@@ -0,0 +1,342 @@
+---
+id: datamodel
+title: Data Model
+sidebar_label: Data Model 
+---
+
+Graphback uses a schema-first approach to generate a GraphQL API and client-side documents using your data models, which are reprented in the GraphQL Schema language as GraphQL types.
```

Comment:
We are model model-first. Schema first do not modify schema. But we also give code based solutions so best to not mention x-first as this can drive people off